extra-libs/libcamera/autobuild/beyond

```diff
@@ -0,0 +1,2 @@
+abinfo 'Fixing the mode of signed IPA modules ...'
+find "$PKGDIR"/usr/lib/libcamera -type f -exec chmod 755 {} +
```

Comment:
Add verbosity here. Also your abinfo doesn't mean much beyond that "I'm doing something here." Say exactly what you did here.